finnish/README.md

```diff
@@ -1,9 +1,26 @@
-# Finish Braille Specifications and Guidelines
+# Finnish Braille Specifications and Guidelines
 
-## [Pistekirjoituksen perusteet](pistek_perusteet.html)
+## [Pistekirjoituksen perusteet (PDF)](Pistekirjoituksen_perusteet.pdf)
 
-- _by_: Finnish Braille Authority and the Finnish Library for the Visually Impaired
+- _description_: 6-dot braille for literary texts
+- _by_: Finnish delegation for braille and Accessibility Library Celia
 - _applies to_: Finland
-- _published_: 2000
-- _language_: Finnish
+- _published_: 2021
+- _language_: Finnish, Finland Swedish
 
+## [Matematiikan, fysiikan ja kemian pistemerkinnät (PDF)](Matematiikan_fysiikan_ja_kemian_pistemerkinnät.pdf)
+
+- _description_: 6-dot braille for math, physics and chemistry texts
+- _by_: Finnish delegation for braille and Accessibility Library Celia
+- _applies to_: Finland
+- _published_: 2022
+- _language_: Finnish, Finland Swedish
+
+## [Matematiikan, fysiikan ja kemian merkinnät elektronisissa oppikirjoissa (PDF)](Matematiikan_fysiikan_kemian_merkinnät_elektronisissa_oppikirjoissa.pdf)
+
+- _description_: 6-dot braille for math, physics and chemistry texts in electronic books (based on ASCIIMath)
```

Comment:
Replace with _title translated_, possibly complemented with a description below (or at the top).